### ISSUE TITLE
Folders path incorrect in documentation

### ISSUE THREAD

**Issue by ruben-alonso** [JiscPER/jper#82]
In the INSTALLATION-ADMINISTRATION guide some of the paths are incorrect. E.g. In the STORE part, when you are explaining how to add the superfisor config you put this command: "sudo ln -s ~/sword/src/sword/deployment/store.conf ." which should be: "sudo ln -s ~/**store**/src/**store**/deployment/store.conf ." This happens a few other times along the document.

**Comment by markmacgillivray** [comment]
Yes. The folder paths are configurable options, and the documentation
states that they are as up to date as possible, but the best way to know
current configuration is to look at the configuration files. You can
actually configure those paths to be anywhere, depending on your
preference.
